### PR TITLE
RelevanceTable

### PR DESCRIPTION
**Pull request by julia-shenshina**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna-ts/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [x] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna-ts/blob/master/CHANGELOG.md)?

## Type of Change

- [ ] Examples / docs / tutorials / contributors update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves an existing feature)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: etna/analysis/feature_relevance/relevance.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from abc import ABC
+from abc import abstractmethod
+
+import pandas as pd
+
+from etna.analysis.feature_relevance.relevance_table import get_statistics_relevance_table
+
+
+class RelevanceTable(ABC):
```
Comment (alex-hse-repository) [inline etna/analysis/feature_relevance/relevance.py:9]: Shell we also inheritance from `BaseMixin`?

File: tests/test_analysis/test_feature_relevance/test_relevance.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from etna.analysis.feature_relevance import StatisticsRelevanceTable
+
+
+def test_statistics_relevance_table(simple_df_relevance):
```
Comment (alex-hse-repository) [inline tests/test_analysis/test_feature_relevance/test_relevance.py:4]: Could you please fix the fixture typing in confest

File: etna/analysis/feature_relevance/relevance.py (hunk 1 above)
Comment (julia-shenshina) [reply etna/analysis/feature_relevance/relevance.py]: fixed

File: tests/test_analysis/test_feature_relevance/test_relevance.py (hunk 2 above)
Comment (julia-shenshina) [reply tests/test_analysis/test_feature_relevance/test_relevance.py:4]: done

**Review by alex-hse-repository — APPROVED** [review]

**Pull request merged**